pyha_analyzer/augmentations.py

```diff
@@ -5,15 +5,55 @@
 import logging
 import os
 from pathlib import Path
-from typing import Any, Callable, Dict, Tuple
+from typing import Any, Callable, Dict, List, Optional, Tuple, Iterable
 
+import numpy as np
 import pandas as pd
 import torch
 import torchaudio
-from pyha_analyzer import config
-from pyha_analyzer import utils
+
+from pyha_analyzer import config, utils
 
 logger = logging.getLogger("acoustic_multiclass_training")
+def invert(seq: Iterable[int]) -> List[float]:
+    """
+    Replace each element in list with its inverse
+    """
+    if 0 in seq: 
+        raise ValueError('Passed iterable cannot contain zero')
+    return [1/x for x in seq]
+
+def hyperbolic(seq: Iterable[int]) -> Dict[float, int]:
+    """
+    Takes a list of numbers and assigns them a probability
+    distribution accourding to the inverse of their values
+    """
+    norm_factor = sum(invert(seq))
+    probabilities = [x/norm_factor for x in invert(seq)]
+    return dict(zip(probabilities, seq))
+
+def sample(distribution: Dict[float, int]) -> int:
+    """
+    Sample single value from distribution
+    distribution.keys() should be the probabilities,
+    and distribution.values() should be the values
+    """
+    probabilities = list(distribution.keys())
+    values = list(distribution.values())
+    return np.random.choice(values, p = probabilities)
+
+def gen_uniform_values(n: int):
```

Comment:
I was gonna add a docstring here but I don't know what the goal of this function is lol